docassemble/mlhframework/data/questions/mlh_interview_framework.yml

```diff
@@ -90,6 +91,12 @@ template: MLH_materials              # default list of needed info. Keep as bull
 content: |
   * Information about your legal issue
   * Information about the people involved
+---
```

Comment:
@ekressmiller - if we put a `mandatory: True` on this block, then `review_screen_prep` doesn't need to be called from each individual interview block. What are your thoughts on that (after vacation, of course)?

Reply:
@normon66 Yes, I like that! Just tested and made a pull request and will request your review. I also tested overriding the messages in an individual interview and that worked fine.